extensions/wikia/Email/Controller/FounderDigestController.class.php

```diff
@@ -172,20 +175,15 @@ protected function assertPageEditsSet() {
 	protected function getDetailsList() {
 		return [
 			[
-				'iconSrc' => ImageHelper::getFileUrl( 'Page-views.png' ),
+				'iconSrc' => ImageHelper::getFileUrl( '100PagesViewed.png' ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "iconSrc" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)